robocorp-code/vscode-client/src/views.ts

```diff
@@ -229,13 +229,7 @@ export class LocatorsTreeDataProvider implements vscode.TreeDataProvider<Locator
             roboCommands.ROBOCORP_GET_LOCATORS_JSON_INFO, { 'robotYaml': robotEntry.robot.filePath });
         if (!actionResult['success']) {
             this.lastRobotEntry = undefined;
-            return [{
-                name: actionResult.message,
-                type: "error",
-                line: 0,
-                column: 0,
-                filePath: robotEntry.robot.filePath,
-            }];
+            return [];
```

Comment:
Why this change? 
-- i.e.: if there's some error loading the info, I think it's reasonable to show it to users...

Reply:
This populates the Locators view with unnecessary tree items, even in "normal" case of not having a locators.json for the robot, or an empty locators.json. Might be better to show errors as notifications maybe?

Reply:
I'd say that if we want to hide and count that as "normal", the language server should return `success:true` with an empty list (so that errors are still shown) and still show errors here.

Reply:
That sounds fair enough, I'll modify those cases so that the language server to respond with success and an empty list

Reply:
Handling now the empty or missing locators.json in language server and showing any errors to user as before